server/service/src/sync/api_v6/mod.rs

```diff
@@ -157,19 +159,22 @@ pub struct SyncPullRequestV6 {
     pub(crate) batch_size: u32,
     pub(crate) sync_v5_settings: SyncApiSettings,
     pub(crate) is_initialised: bool,
+    pub(crate) sync_v6_version: u32,
```

Comment:
I think we need to set this to default to 1. Would be good to test with 2.0 omSup remote to 2.1 omSup remote (would suspect there would be serde error,  unless default is set)

Reply:
Ah yeah you're right - is this the best way to implement that? Feels a bit icky? Maybe should have a sync_v6_settings struct instead so default is only set in one place??

https://github.com/msupply-foundation/open-msupply/compare/3111-oms-central-sync-versioning...3111-oms-central-sync-versioning(default-sync-version)

Reply:
> Ah yeah you're right - is this the best way to implement that? Feels a bit icky? Maybe should have a sync_v6_settings struct instead so default is only set in one place?? Or maybe there is a better way in general lol
> 
> [3111-oms-central-sync-versioning...3111-oms-central-sync-versioning(default-sync-version)](https://github.com/msupply-foundation/open-msupply/compare/3111-oms-central-sync-versioning...3111-oms-central-sync-versioning(default-sync-version))